metrics/generatecommands/main.go

```diff
@@ -0,0 +1,105 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/docker/compose-cli/utils"
+)
+
+var managementCommands = []string{"ecs", "assemble", "registry", "template", "cluster", "scan"}
```

Comment:
IIRC all of these except `ecs`, and `scan` were only ever in EE so we could remove them.

Reply:
Indeed, added here because I don't get them with Desktop, it's enterprise. I kept these because they were already listed as management commands , so we can at least know if some users are calling them (we won't get details with subcommands as I have not generated subcommands for `template` for example). 
@nebuk89, OK to just ignore assemble, registry, template, cluster ?

Reply:
Yep, thats fine with me!

Reply:
Done